pkg/parser/model/model.go

```diff
@@ -2019,17 +2020,25 @@ func (t RunawayActionType) String() string {
 		return "COOLDOWN"
 	case RunawayActionKill:
 		return "KILL"
+	case RunawayActionSwitchGroup:
+		return "SWITCH_GROUP"
 	default:
 		return "DRYRUN"
 	}
 }
 
+// RunawayActionSettings is the settings of the runaway action.
+type RunawayActionSettings struct {
+	Type            RunawayActionType `json:"type"`
+	SwitchGroupName string            `json:"switch_group_name"`
+}
+
 // ResourceGroupRunawaySettings is the runaway settings of the resource group
 type ResourceGroupRunawaySettings struct {
-	ExecElapsedTimeMs uint64            `json:"exec_elapsed_time_ms"`
-	Action            RunawayActionType `json:"action"`
-	WatchType         RunawayWatchType  `json:"watch_type"`
-	WatchDurationMs   int64             `json:"watch_duration_ms"`
+	ExecElapsedTimeMs uint64                `json:"exec_elapsed_time_ms"`
+	Action            RunawayActionSettings `json:"action"`
```

Comment:
Does changing the json type break compatibility?

Reply:
Great catch, I forgot to handle the compatibility issue here.